Makefile

```diff
@@ -34,8 +34,9 @@ endif
 
 # installation directories (for 'make install')
 prefix=/usr/local
-libdir=$(prefix)/lib
-includedir=$(prefix)/include
+libdir=lib
```

Comment:
In the [GNU-style installation variables](https://www.gnu.org/prep/standards/html_node/Directory-Variables.html), `libdir` should be the whole path, defaulting to `$(prefix)/lib`.   So the previous version of this code was correct.